### DISCUSSION TITLE
Android 4 (and posting on github discussions using v86)

### DISCUSSION THREAD

**Discussion by hello-smile6** [copy/v86#575] [General]
@copy recently added Android 4 to v86's default profiles, but didn't give an announcement. I'm going to (if I can) log into GitHub and make a post from v86. (It's my longtime obsession, doing that)
https://copy.sh/v86/?profile=android4 !

**Comment by hello-smile6** [comment]
v86 has signed into github! I repeat, v86 has signed into github! 🎉 
![image](https://user-images.githubusercontent.com/73048226/146493198-bc23b31e-4256-4364-9857-8a1c10f5a8ff.png)

**Comment by hello-smile6** [comment]
v86 has entered the DISCUSSION! We're so close to the first GitHub post made from v86!!!
![image](https://user-images.githubusercontent.com/73048226/146493514-97e48709-870f-4724-8667-38224fbfc610.png)

**Comment by hello-smile6** [comment]
Writing a post!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! 🎉 
![image](https://user-images.githubusercontent.com/73048226/146494071-45406085-d566-4a53-8f3e-67f074eb5b9a.png)

**Comment by hello-smile6** [comment]
Hi from v86. v86 has officially posted on the v86 GitHub repo!!! :tada:

**Reply by hello-smile6** [reply]
Hi from MSEdge Desktop! I hereby welcome v86 as an emulator with proportional capabilities to QEMU!!!!

**Comment by hello-smile6** [comment]
v86 again. Thanks for the celebration, but I have to give some credit to the Android, Android-x86, and Chromium developers who made this possible. And, of course, to the v86 devs.

**Reply by hello-smile6** [reply]
Yep, that's v86. 
![image](https://user-images.githubusercontent.com/73048226/146495043-57f8111b-f5a1-4b5e-a543-88e335064bb8.png)

**Comment by hello-smile6** [comment]
Next: v86 on v86!!!

**Reply by hello-smile6** [reply]
Never mind 😦 Not going to work. https://en.wikipedia.org/wiki/WebAssembly says WASM is 3 years newer than that version of Chromium. I'm off to post on personal things!

**Reply by porg13231** [reply]
if it worked i would like to see v86 on v86 on v86...

**Reply by raspiduino** [reply]
@hello-smile6 get a Buildroot with Firefox on it. Then run v86 :)

**Reply by hello-smile6** [reply]
@raspiduino .iso file?

**Reply by raspiduino** [reply]
You can build one yourself. It is pretty easy.

**Reply by hello-smile6** [reply]
@raspiduino I can't. I lack both a development environment and the time to build one.

**Comment by leap0x7b** [comment]
calm down you dont need to be so excited

**Reply by hello-smile6** [reply]
Why?

**Comment by raspiduino** [comment]
If you are lack of a development environment, use [Google Cloud Shell](
https://shell.cloud.google.com) for free. If you are lack of time, I will
build and test one for you.

**Reply by hello-smile6** [reply]
@raspiduino Wait, I can build on there? I've used that before. Let me try...

**Comment by raspiduino** [comment]
Why not? It's a Linux environment with root. Do anything you want (except
for illegal things)

**Reply by hello-smile6** [reply]
Okay, what packages do I need to get a working system for v86?

**Comment by raspiduino** [comment]
Go to buildroot.org and there will be tutorials for you. Or you can do a
google search, pretty easy :)

**Reply by hello-smile6** [reply]
Okay

**Reply by hello-smile6** [reply]
@raspiduino Is it seriously a free machine with 8 GB of memory?

**Comment by raspiduino** [comment]
Yes. But the limitation is you can only use it for maximum 40 hrs a week
and you only get 15 gb of disk

**Reply by hello-smile6** [reply]
@raspiduino So, if I only use it for 5 hours each day, I'll never run into issues.

**Reply by hello-smile6** [reply]
@raspiduino I built a system, what do I do now?

**Reply by raspiduino** [reply]
Did you get the Buildroot iso?

**Reply by hello-smile6** [reply]
@raspiduino Have a rootfs tarball
